runtime/bcutil/ROMClassBuilder.cpp

```diff
@@ -1059,17 +1059,17 @@ ROMClassBuilder::finishPrepareAndLaydown(
  *                                  + UNUSED
  *                                 + UNUSED
  *                                + UNUSED
- *                               + UNUSED
+ *                              + J9AccHiddenClassOptionNestmate
```

Comment:
I would prefer these new flags follow the convention - start with J9AccClass

Reply:
Fixed.